dlls/ff/ff_lualib_team.cpp

```diff
@@ -32,6 +33,29 @@ std::ostream& operator<<(std::ostream& stream, const CTeam& team)
 	return stream << "team" << ":" << const_cast<CTeam&>(team).GetName() << ":" << const_cast<CTeam&>(team).GetTeamNumber();
 }
 
+namespace FFLib
+{
+	// helper function to turn the bitmask into a lua table of teams
+	luabind::adl::object GetAllies(CFFTeam *pTeam)
+	{
+		luabind::adl::object luatblAllies = luabind::newtable(_scriptman.GetLuaState());
+		if (pTeam)

```

Comment:
Just realized that it's actually impossible for pTeam to be NULL here. Lua exposes the function as CFFTeam::GetAllies, so there's no way to call it with a NULL object.